docker/development/standup.sh

```diff
@@ -4,6 +4,6 @@ BLOCKTIME=${BLOCKTIME:-3}
 
 echo "using block time of ${BLOCKTIME} seconds"
 
-(node /standup/prepare-ganache-for-unlock.js &)
+# (node /standup/prepare-ganache-for-unlock.js &)
```

Comment:
I like this approach but the docker image is used by more than just unlock-js (for exaple it is the one used for local dev). It is important that we have a way to run this script in more than just unlock-js

Reply:
Agreed, this is currently in draft to shake things out. This looks like it will be a large change, id like to see if we can reach a state where the following occur:

1. The test manages its stand up/tear down
2. The development environment exists as it currently does
3. We untangle these concerns so that that they can develop independently. ie: the unlockjs test suite should be able to test an erc20 backed lock that doesnt currently exist in the development setup